### DISCUSSION TITLE
Reduction of degrees of freedom

### DISCUSSION THREAD

**Discussion by VForiel** [VForiel/Tunable-Kernel-Nulling#31] [Ideas]
Some degree of freedom seems redoundant, such as $P_1$ which can be removed and use the first input's phase as the reference phase. We can also consider removing $P_9$, $P_{10}$ and $P_{12}$ for the same reason. Does it work as expected? Can we also consider removing $P_3$, $P_5$ and $P_6$?

![dof_reduction](https://github.com/Leirof/Tunable-Kernel-Nulling/assets/12165342/9e60fbb3-20f4-4927-89fe-382042da467c)

**Comment by rlaugier** [comment]
That's an interesting question. At first glance, it looks to me that you will have to keep seven independent actuators.
But overall, I think that it might be time to think about these in terms of modes, the same way we sometimes like Zernike modes or Fourier modes. Indeed, you might discover that it is better to consider for example [(P1-P2), (P3-P4), (P1+P2) - (P3+P4), ...] which would constitute a control matrix...
